### ISSUE TITLE
Faire connaître l'idée et l'existence de la réflexion

### ISSUE THREAD

**Issue by kryzaal** [ordre-des-developpeurs/odd-constitution#6]
Quelles actions concrètes ? Quelle communication ? Comment amener des développeurs à prendre part à cette réflexion ?

**Comment by BodySplash** [comment]
Bonne question. 

Pour le moment, mon idée est assez simpliste : tenter de faire le tour du plus de conférences possible pour parler du concept, pour inviter les gens ensuite à animer des assemblées locales pour faire avancer le bousin. Effectivement ça ne va attirer dans un premier temps que des développeurs déjà assez impliqués, mais ça me paraît une bonne étape avant de s'attaquer à la masse de développeurs «dormants». Cette approche peut bien entendu être utilisée par tout le monde, pas seulement moi, surtout que je vais avoir du mal à faire toutes les villes de France :) 

Nous avons dans les stocks également d'ouvrir un slack. En fait techniquement, il est déjà ouvert, mais il faudrait faire une page ou quelque chose pour permettre aux développeurs de le rejoindre sans passer par la case invitation. L'avantage de Slack, c'est que j'ai observé qu'il crée assez facilement une certaine cohésion entre les participants. 

Ensuite je suppose qu'il y a les outils classiques à mettre en œuvre : twitter, site web, etc.

**Comment by SetaSensei** [comment]
Je rajoute juste mon petit grain de sel : j'ai écris un petit post sur le blog de ma boîte, histoire de commencer quelque part : http://blog.softit.fr/post/agile-france-la-naissance-d-un-ordre-des-developpeurs
Le concept a plus à pas mal de collègues, on s'est dit que c'était déjà une première idée.

Quand tu parles d'un site, tu veux en faire plutôt une vitrine de l'idée ou finalement y rapatrier tout ce qu'il y aura sur slack et / ou github ?

**Comment by JeffLeFoll** [comment]
Je rejoints Jean-Baptiste, la tournée des conférences peut etre un très bon point de départ justement pour former une base de développeurs qui s'implique déjà dans certaines communautés.

Là on arrive à la "trève" estivale mais en plus des conférences il y a aussi les JUG et autres regroupement de développeurs. 

Les agiles tours arrivent courrant octobre / novembre on pourrait essayer de préparer quelques choses et que ceux qui le veulent interviennent dans leur édition locale ?

**Comment by kryzaal** [comment]
Je suis sur Paris l'an prochain, cela peut être utile.

**Comment by lilobase** [comment]
Hello, pour le slack, le système d'invitation automatique est disponible : http://odd-slack.herokuapp.com/

**Comment by ophir82** [comment]
Bonjour à tous !

Histoire d'ajouter ma petite pierre, juste une idée qui a son petit succès dans notre boîte : faire des opérations "kata + pizzas" entre midi et deux.
Là ça reste confiné au sein de l'entreprise, mais le format fonctionne bien et je me dis qu'il pourrait aussi s'étendre en dehors ce cadre.

**Comment by BodySplash** [comment]
Effectivement, c'est le format brown bag launch : http://www.brownbaglunch.fr
Ça pourrait valoir le coup d'ajouter des présentations de l'ordre sur le site. Je pense que je vais l'ajouter pour moi même en tout cas. 

@lilobase ton code est dispo quelque part ? L'idée serait  de monter un petit site de présentation avec ça + le formulaire d'inscription, avec une case à cocher «rejoindre le slack»

Pour les Agile Tours, je pense que c'est quasi sûr pour moi pour Bordeaux et Toulouse, et j'attend d'avoir la réponse de Lille.

**Comment by JeffLeFoll** [comment]
Pour les Agile tours, le CFP de celui de Brest n'est pas encore ouvert. Cela peut me laisser du temps pour préparer quelque chose et leurs soumettre.
Je pourrais ensuite également référencer cette présentaiton sur le site du brownbaglunch.fr.

**Comment by lilobase** [comment]
@BodySplash j'utilise un petit truc en nodejs : https://github.com/outsideris/slack-invite-automation

**Comment by kryzaal** [comment]
@BodySplash : Je me lancerais bien pour d'autres villes, mais je ne suis pas un orateur, hélas. Si personne n'est disponible, je peux me dévouer pour une ville autre que Paris (trop gros).

Bonnes idées le brown bag lunch et le slack.

**Comment by ophir82** [comment]
Je connaissais pas brown bag lunch mais effectivement ça colle pas mal !

Arriver dans une entreprise en parlant d'"ordre", ça peut peut-être faire peur, non ?

D'autre part, je ne sais pas trop comment un brown bag lunch s'organise, mais il faudra peut-être valider le public dans un premier temps pour pouvoir s'adresser à des gens sensibilisés aux bonnes pratiques.

**Comment by BodySplash** [comment]
Juste pour information, du coup je vais animer le premier bbl parlant de l'ordre demain. Je suis curieux de voir l'accueil en enterprise…

**Comment by kryzaal** [comment]
J'attends ton retour avec impatience !

**Comment by Squallygator** [comment]
moi aussi ;)

**Comment by ophir82** [comment]
J'étais là, du coup je peux peut-être faire un retour de ce qui s'est dit après le bbl à la machine à café :
- phase 1 : bon feedback et bonne énergie chez les développeurs qui ont assistés à la présentation ;
- phase 2 : très rapidement, « ouais mais dans notre boîte, on pourra jamais appliquer ça », «tu imagines la tête de la hiérarchie si essaie d'imposer d'autres pratiques » ;
- phase 3 : tout le monde n'applique pas les bonnes pratiques (juniors dans l'équipe), il faudrait négocier pas mal de temps de formation, pair programming, etc. Ce qui n'est pas dans les habitudes de la boîte.
- phase 4 : « bon on retourne bosser, t'as fait des tests, toi  » ?

Au final, une belle petite étincelle. Pas évident à concrétiser par la suite.
